.gitignore

```diff
@@ -486,7 +486,6 @@ dmypy.json
 .pyre/
 
 # build
-.vscode/
```

Comment:
I would keep this dir in the `.gitignore` file, so changes in these files are ignored, unless a user explicitly adds them with `git add .vscode/ --force`.